video/bink_decoder.cpp

```diff
@@ -76,8 +76,8 @@ bool BinkDecoder::loadStream(Common::SeekableReadStream *stream) {
 	if ((id != kBIKfID) && (id != kBIKgID) && (id != kBIKhID) && (id != kBIKiID))
 		return false;
 
-	uint32 fileSize         = stream->readUint32LE() + 8;
-	uint32 frameCount       = stream->readUint32LE();
+	uint32 fileSize = stream->readUint32LE() + 8;
```

Comment:
We want to leave this to developers. E.g. to not force alignment or not-alignment.

Reply:
That would be hard to determine what was intended vs unintended - if we want to improve code formatting in other areas this might be a place where we need to be more lax.